### PR TITLE
Add test file for DataFormats/L1Scouting

### PR DESCRIPTION
**Pull request by Mmiglio**
Add initial data files that will be used in a unit test of the L1 Scouting data formats. These will be read by future releases to verify backward compatibility.

This is related to https://github.com/cms-sw/cmssw/pull/43467. I will open a new PR with improved unit tests for data format.

### CODE REVIEW

**Comment by smorovic** [comment]
please test

**Comment by smorovic** [comment]
+daq
(tests still running, but I guess new root file shouldn't in fact have effect)

**Comment by smuzaffar** [comment]
+externals

**Comment by Mmiglio** [comment]
Hi all, is there anything missing for this PR to be merged? 
I assume I need to wait for this PR to be merged before adding the test to CMSSW.
The same question applies to https://github.com/cms-data/DataFormats-L1ScoutingRawData/pull/1.

**Comment by smuzaffar** [comment]
only new data file, so I am merging it now

File: testL1Scouting_13_3_0_pre5.root
Comment (makortel) [inline testL1Scouting_13_3_0_pre5.root:1]: To follow the naming convention in `DataFormats/Scouting`, please rename the file as `testL1Scouting_v3_v3_v3_v3_v3_13_3_0_pre5.root` to include the versions of each the classes. The order should reflect the order of the classes in the `DataFormats/L1Scouting/src/classes_def.xml`, which I'd also ask to be reordered in alphabetical order in the PR that adds the tests using this file.

**Pull request merged**